usort/tests/cli.py

```diff
@@ -33,6 +33,19 @@ def sample_contents(s: str) -> Generator[str, None, None]:
 
 
 class CliTest(unittest.TestCase):
+    def test_benchmark(self) -> None:
+        with sample_contents("import sys\n") as dtmp:
+            runner = CliRunner()
+            with chdir(dtmp):
+                result = runner.invoke(main, ["--benchmark", "check", "."])
+
+        self.assertIn("walking .:", result.output)
```

Comment:
Can you make this a little more exact, maybe use a regex to replace <looks like a float> with `#` and use assertEqual on the entire output?